test/dlc_tests/sanity/test_utility_installation.py

```diff
@@ -84,6 +104,28 @@ def test_utility_packages_using_import(training):
             assert Version(version) > Version(
                 "2"
             ), f"Sagemaker version should be > 2.0. Found version {version}"
+            
+
+@pytest.mark.model("N/A")
+@pytest.mark.integration("common pytorch training utility packages")
+def test_commmon_pytorch_utility_packages_using_import(pytorch_training):
```

Comment:
Please also add `@pytest.mark.usefixtures("sagemaker")` to make this test run on SM DLCs.